app/routes/application.js

```diff
@@ -66,9 +66,14 @@ export default Route.extend({
   },
 
   updateProperty(options) {
+    if (!this.get('controller.mixinDetails.mixins')) return;
```

Comment:
Rather than this early return, could we wrap the rest of the function in  `if (this.get('controller.mixinDetails.mixins'))`?